### ISSUE TITLE
Review log levels and warinings

### ISSUE THREAD

**Issue by tvoinarovskyi** [aio-libs/aiokafka#461]
As #335 issue and some related threads (https://github.com/ManageIQ/aiops-incoming-listener/pull/11) point out, current logging levels are not exactly the best and cause some confusion. Will need to go over Java's log levels for most critical errors including heartbeats and set aiokafka's levels accordingly.

**Comment by bafitor** [comment]
Any updates regarding this?

**Comment by tvoinarovskyi** [comment]
Not really. Did not get a chance to do it yet. It'd not too hard, PR welcome. If you have any logs that bother you, sharing them would also help.
